### ISSUE TITLE
Receipts Not Being Sent to Customers

### ISSUE THREAD

**Issue by marksull** [202-ecommerce/stripe_official#31]
After a successful charge, the Stripe Dashboard indicates in the `Receipt history` section for each Payment:

```
No receipts sent
No email address for this payment
```

While I see that you are sending the `Billing Details` > `email` in the charge payload, according the the Stripe API docs, for an email to be sent to the customer the field `receipt_email` must be provided in the Payment Intent.

https://stripe.com/docs/api/payment_intents/create#create_payment_intent-receipt_email

Is this an enhancement that can be included in a future version?

**Comment by clotaire202** [comment]
Dear @marksull 

Thank you for your suggestion.We will study it with Stripe

**Comment by marksull** [comment]
While indeed Prestashop does send email, having the Payment Gateway (Stripe in this case) send an official receipt from their service helps align the experience with what customer currently receive from other Payment Gateways (like PayPal).

**Comment by mrabsinthe** [comment]
I agree with @marksull , receiving receipts from other payment gateways incl. PayPal is a common practice. It would be therefore great if this option is included.

**Comment by 202-ecommerce** [comment]
After years of hard work with Stripe connector for PrestaShop, 202 ecommerce will stop supporting Stripe module on January 9th 2023.
After this date, merchants will get [support directly](https://support.stripe.com/contact/email?topic=third_party_integrationshttps://support.stripe.com/contact/email?question=other&topic=third_party_integrations&subject=Prestashop%20Support%20Request&body=Include%20details%20of%20your%20Prestashop%20issue:%20)
